### DISCUSSION TITLE
Advertising for the best distro there is...

### DISCUSSION THREAD

**Discussion by AlphaElwedritsch** [SpiralLinux/SpiralLinux-project#395] [General]
I've been campaigning for the last few days.
And some people can be hooked on the advantages of Spirallinux.
It would be nice if it showed up in the download numbers.

**Comment by ghost** [comment]
Why?  SpiralLinux is like a sleeper hot rod - slim beauty in the outside but plenty of muscle on the inside for those demanding tasks  :P  - I mean, it is Debian inside, you know?  But without all the Debian messy nonsense  :P

**Comment by banzaigtv** [comment]
I have SpiralLinux Budgie on one of my external SSDs and I must testify that it is perhaps the best version of SpiralLinux and one of the best distros out there. Qt apps seamlessly use the GTK themes and even the Dropbox app menu uses the dark themes. I'm hooked on Plasma (Debian 12), but if necessary, SpiralLinux Budgie will be the computer's next daily driver. This project needs to be revealed and talked about more on YouTube, Facebook, etc.

**Reply by ghost** [reply]
Yes, but while I have never been a Budgie fan, I have always liked Debian and most derivatives.  SpiralLinux has been the best Stable version for me and all my varied systems - an Old E1 Vision, and few i5s, a Ryzen 7, and a very old Apple cMP 5,1 (a Mac Pro desktop, monster) ... but on the cMP I am forced to use 'nomodeset' or the video refuses to work  :(   lol
